modules/ROOT/pages/index.adoc

```diff
@@ -29,15 +29,35 @@ This leads to some key benefits:
 ** Elevated privileges required to provision resources are granted to the Service Broker, not end users
 ** Resources can provisioned in a way where they are hidden from the end user
 
-== What does the Service Broker do?
+== What does the Couchbase Service Broker do?
 
-This implementation of an Open Service Broker works exclusively with Kubernetes resources.
+The Couchbase Service Broker works exclusively with Kubernetes resources.
 Creating service instances and bindings creates Kubernetes resources.
-This is done in a highly abstracted way so the Service Broker can work with any Kubernetes resource type.
+This is done in a highly abstracted way so that the Couchbase Service Broker can work with any Kubernetes resource type.
 
 Resources are templated, so they may be dynamically configured based on a set of inputs.
 These inputs include metadata--provided by the client to provision the instance--and user specified parameters.
 
+image::csb-overview-diagram.png[]
+
+. Let’s say we have a Service Broker created and deployed on the Kubernetes platform.
+
+. Register the Service Broker with the platform.
+Once the Service Broker is registered with the platform (Kubernetes), users can view the service offerings by listing all the services and service plans provided by different Service Brokers through the service catalog.
```

Comment:
It's registered with the service catalog, which is just another operator, nothing to do with the platform.  Unless platform is the service catalog platform and not the kubernetes platform... or the platform is an aggregate of kubernetes and the service catalog.

Reply:
I'll get your diagram in there some way or another. Promise